kubeflow/core/update_backend.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# A simple shell script to configure the backend timeouts and health checks by using gcloud.
+[ -z ${NAMESPACE} ] && echo Error NAMESPACE must be set && exit 1
+[ -z ${SERVICE} ] && echo Error SERVICE must be set && exit 1
+
+PROJECT=$(curl -s -H "Metadata-Flavor: Google" http://metadata.google.internal/computeMetadata/v1/project/project-id)
+if [ -z ${PROJECT} ]; then
+echo Error unable to fetch PROJECT from compute metadata
+exit 1
+fi
+
+# Activate the service account
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
```

Comment:
I think we need to add retries here:
https://github.com/kubeflow/kubeflow/blob/7a84feb3f0caf3bf7c282c31008073d90edceaf0/kubeflow/core/configure_envoy_for_iap.sh#L22

To allow for the fact that service account might not work immediately.